KaMProjectGroup.groupproj

```diff
@@ -1,264 +1,276 @@
-﻿	<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
Can you revert this file and DPROJ too please? The changes in them are irrelevant and just clutter the PR =)